### ISSUE TITLE
A11y_SQLAzureDataStudioMAC_Dashboard_VoiceOver: The Voice-over announces, 'Learn More' link as button inside the 'Dashboard' section.

### ISSUE THREAD

**Issue by bbarik06** [microsoft/azuredatastudio#19880]
**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev2;#BM_SQLAzureDataStudioMAC_June2022;#SQL Azure Data Studio;#MAC;#FTP;#DesktopApp;#VoiceOver;#AINotInScope;#WCAG4.1.2;

## Environment Details:
Application Name: SQL Azure Data Studio
Application Version: Version 1.37.0
Mac Version: macOS Monterey 12.4
Screen Reader: Voice-Over

## Repro Steps:

1. Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility tester after verification of the fix.
2. Launch SQL Azure data studio.
3. Connect to a server.
4. Right click on the server and select 'Manage'.
5. Dashboard will open.
6. Tab till 'Learn More' control.
7. Turn on Voice-over and observe.

## Actual:
The Voice-over announces, 'Learn More' link as button inside the 'Dashboard' section.

## Expected:
The Voice-over should announce 'Learn More' as links since it leads the user to a web page destination.

## User Impact:
Screen readers users may find it difficult to perceive the user interface component if incorrect role is assigned.

## Attachment:
![19_Learn more](https://user-images.githubusercontent.com/98393966/176434254-71aa29fc-ad7a-4004-bf4f-3796c8726e2e.gif)

**Comment by alanrenmsft** [comment]
this is by design. The button's handler is to open a web page, it doesn't necessarily make itself a link.
